incubator/hnc/Makefile

```diff
@@ -60,8 +60,12 @@ CURDIR = $(shell pwd)
 KUSTOMIZE ?= ${CURDIR}/hack/kustomize-3.8.1
 CONTROLLER_GEN ?= ${CURDIR}/bin/controller-gen
 
-# Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
-CRD_OPTIONS ?= "crd:trivialVersions=true"
+# Enable CRD per-version validation. As of Kubernetes 1.13, we can have multiple
+# versions of the Kind defined in the CRD, and use a webhook to convert between
+# them. By default, KubeBuilder disables generating different validation for
+# different versions of the Kind in the CRD, to be compatible with older
+# Kubernetes versions.
+CRD_OPTIONS ?= crd
```

Comment:
If the default is trivial, don't we need "trivialVersions=false" here?

Reply:
According to https://book.kubebuilder.io/reference/generating-crd.html#multiple-versions, it should be changed to `CRD_OPTIONS ?= crd`. I tested both and `CRD_OPTIONS ?= crd` and `CRD_OPTIONS ?= "crd:trivialVersions=false` are the same.

Reply:
Hmm, I find the second version easier to read - it's unclear to a casual reader what "CRD options is 'crd'" means. Maybe use the more explicit version or explain that this funny syntax means "force all options to false" or something, and link to the book?

Reply:
SG, updated with `CRD_OPTIONS ?= "crd:trivialVersions=false` and tested.